pkg/controller/core/clusterqueue_controller.go

```diff
@@ -299,6 +315,8 @@ func (r *ClusterQueueReconciler) Create(e event.TypedCreateEvent[*kueue.ClusterQ
 		log.Error(err, "Failed to add clusterQueue to queue manager")
 	}
 
+	r.createDefaultLocalQueues(ctx, e.Object)
```

Comment:
How about `Update()` - should we react e.g. to:

- switching `DefaultLocalQueue` from `nil` to non-`nil`
- expanding the namespace selector of a CQ? 

Or are all these bits of spec immutable?